packages/story-editor/src/components/dropTargets/karma/dropTarget.karma.js

```diff
@@ -100,7 +100,7 @@ describe('Drop-Target integration', () => {
       expect(rep2).toBeEmpty();
 
       // Verify the background base color is handled as expected.
-      expect(bgElement.resource.baseColor).toEqual([201, 201, 219]);
+      expect(bgElement.resource.baseColor).toEqual('#060606');
```

Comment:
Where did `[201, 201, 219]` come from before? Where is `#060606` coming from now?

I want to make sure that this is not actually a regression.

Reply:
I updating the fixture data seems to have fixed this issue. You have to remember, this is a change of behaviour now. Base color will be generated on update, meaning it will available. So fixture data needed to be updated.

Reply:
The point of this check was previously to ensure that when media without base color is added to the page, it's base color gets calculated correctly.

Since now it's just preset in the fixture, there's no check for that, it just uses the color from the preset values in the fixture.

There's still a possibility for the base color not to exist beforehand, right? Shouldn't it be calculated then when adding the media from library, perhaps the test could check for that now?

Reply:
Yeah, it is possible for the base color not to exist. But the behaviour has now changed. If an media is inserted without base color, then it is generated async and all elements using that resource are updated. We are trying to avoid generating base color on drop, as it is bad for performance.

Reply:
> We are trying to avoid generating base color on drop, as it is bad for performance.

Yep, I know that, I'm basically asking if it's still possible to test the current new logic in a Karma test as well. To ensure that if the base color does not exist, it's eventually added.

Reply:
I have added this test. https://github.com/google/web-stories-wp/pull/9638/commits/55a992837fd637fcd40459931cfedc50ae3c2bc9 . The fixture data does not contain color for this image. Meaning the color is successfully generated.